one-ui/src/main/java/com/marklogic/hub/oneui/auth/MarkLogicAuthenticationManager.java

```diff
@@ -90,62 +85,41 @@ public Authentication authenticate(Authentication authentication) throws Authent
         try {
             restTemplate.getForObject(uri, String.class);
             hasManagePrivileges = true;
-        }
-        catch(ResourceAccessException ex) {
-            throw new BadCredentialsException("Cannot connect to MarkLogic at " + environmentInfo.mlHost + ". Are you sure MarkLogic is running?");
-        }
-        catch(HttpClientErrorException ex) {
+        } catch (ResourceAccessException ex) {
+            throw new BadCredentialsException(
+                "Cannot connect to MarkLogic at "
+                    + environmentInfo.mlHost
+                    + ". Are you sure MarkLogic is running?");
+        } catch (HttpClientErrorException ex) {
             if (HttpStatus.UNAUTHORIZED.equals(ex.getStatusCode())) {
                 throw new BadCredentialsException("Unauthorized");
-            } else if (!(HttpStatus.NOT_FOUND.equals(ex.getStatusCode()) ||
-               HttpStatus.FORBIDDEN.equals(ex.getStatusCode())
-            )){
-                /*
-                    throw error if not NOT_FOUND or FORBIDDEN, as those errors mean proper credentials,
-                    but no access to manage API
-                 */
+            } else if (!(HttpStatus.NOT_FOUND.equals(ex.getStatusCode())
+                || HttpStatus.FORBIDDEN.equals(ex.getStatusCode()))) {
+        /*
+           throw error if not NOT_FOUND or FORBIDDEN, as those errors mean proper credentials,
+           but no access to manage API
+        */
                 throw ex;
             }
         }
         DatabaseClient dataServicesClient = hubConfig.newStagingClient(null);
         boolean stagingServerAccessible = false;
         try {
-            stagingServerAccessible = dataServicesClient.checkConnection().isConnected();
+            stagingServerAccessible =
+                dataServicesClient.newServerConfigManager().getQueryValidation() != null;
```

Comment:
Why is this being used instead of "isConnected"? I believe checkConnection was added for that exact purpose.

Reply:
Oh, my change was based on an older version. I will delete the PR and recreate a new one.